### PR TITLE
Remove useless variables

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Some minor change I made when commenting anki's code. In case you're interested in removing some variables used only once which does not had much readability

### CODE REVIEW

File: qt/aqt/mpv.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def _reader(self):
         while not self._stop_event.is_set():
             if isWin:
                 try:
-                    (n, b) = win32file.ReadFile(self._sock, 4096)
+                    (n, b) = win32file.ReadFile(self._sock, 4096)[1]
```
Comment (dae) [inline qt/aqt/mpv.py:235]: This looks incorrect

Comment (Arthur-Milchior) [reply qt/aqt/mpv.py:235]: I first removed the n, then while re reading I thought that it's more clear to leave it this way, even if `n` is never used, so I added it back. Not correctly, sorry

**Comment by dae** [comment]
Will merge after 2.1.28 release

**Comment by Arthur-Milchior** [comment]
Ok. Thanks

**Pull request merged**